webextensions/api/declarativeNetRequest.json

```diff
@@ -1201,6 +1201,25 @@
               },
               "safari_ios": "mirror"
             }
+          },
+          "filter": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "111"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "127"
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
```

Comment:
@willdurand do we have any information on Safari implementation?

Reply:
Hey @xeenon, can you help here? Thanks!

Reply:
@willdurand @rebloor Safari does not support `filter` currently.

Reply:
Thanks @xeenon